src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java

```diff
@@ -0,0 +1,206 @@
+import java.nio.file.Path;
+import java.sql.*;
+import java.util.*;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Level;
+
+import com.azure.storage.blob.BlobContainerClient;
+import com.exasol.adapter.document.files.abstestsetup.LocalAbsTestSetup;
+import com.exasol.exasoltestsetup.ServiceAddress;
+import org.jetbrains.annotations.NotNull;
+
+import com.exasol.adapter.document.UdfEntryPoint;
+import com.exasol.adapter.document.files.abstestsetup.AbsTestSetup;
+import com.exasol.bucketfs.Bucket;
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.dbbuilder.dialects.DatabaseObject;
+import com.exasol.dbbuilder.dialects.exasol.*;
+import com.exasol.dbbuilder.dialects.exasol.udf.UdfScript;
+import com.exasol.exasoltestsetup.ExasolTestSetup;
+import com.exasol.udfdebugging.UdfTestSetup;
+
+import jakarta.json.*;
+import lombok.Getter;
+import lombok.Setter;
+
+public class IntegrationTestSetup implements AutoCloseable {
+    private static final String ADAPTER_JAR = "document-files-virtual-schema-dist-6.0.1-azure-blob-storage-1.0.0.jar";
+    private final ExasolTestSetup exasolTestSetup;
+    private final Connection exasolConnection;
+    private final Statement exasolStatement;
+    private final ExasolObjectFactory exasolObjectFactory;
+    private final AdapterScript adapterScript;
+    private final Bucket bucket;
+    private final List<DatabaseObject> createdObjects = new LinkedList<>();
+    private final AbsTestSetup absTestSetup;
+    private final BlobContainerClient absContainer;
+    private final UdfTestSetup udfTestSetup;
+    @Getter
+    @Setter
+    private ConnectionDefinition connectionDefinition;
+
+    public IntegrationTestSetup(final ExasolTestSetup exasolTestSetup, final AbsTestSetup absTestSetup,
+            final BlobContainerClient absContainer)
+            throws SQLException, BucketAccessException, TimeoutException, FileNotFoundException {
+
+        this.absTestSetup = absTestSetup;
+        this.absContainer = absContainer;
+
+        this.exasolTestSetup = exasolTestSetup;
+        this.exasolConnection = this.exasolTestSetup.createConnection();
+        this.exasolStatement = this.exasolConnection.createStatement();
+        this.exasolStatement.executeUpdate("ALTER SESSION SET QUERY_CACHE = 'OFF';");
+
+        this.bucket = this.exasolTestSetup.getDefaultBucket();
+        this.udfTestSetup = new UdfTestSetup(this.exasolTestSetup, this.exasolConnection);
+
+        final List<String> jvmOptions = new ArrayList(Arrays.asList(this.udfTestSetup.getJvmOptions()));
+        this.exasolObjectFactory = new ExasolObjectFactory(this.exasolConnection,
+                ExasolObjectConfiguration.builder().withJvmOptions(jvmOptions.toArray(String[]::new)).build());
+        //we create a schema called adapter
+        final ExasolSchema adapterSchema = this.exasolObjectFactory.createSchema("ADAPTER");
+        //create a connection to Azure Blob Storage
```

Comment:
```suggestion
```
use this method name onstead of a comment here

Reply:
reads easier for me this way